internal/controller/dispatcher.go

```diff
@@ -202,9 +215,11 @@ func (r *Dispatcher) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		case mcadv1beta1.Deleted:
 			if r.MultiClusterMode {
 				// TODO: Multicluster. This is where we delete the placement that maps
```

Comment:
Update comment to remove TODO (or remove entirely if you think the code doesn't need a comment anymore)